settings.js

```diff
@@ -5,14 +5,36 @@ const StackUtils = require('stack-utils')
 
 // Just unconditionally use fs.rmdirSync after LTS/12 is required
 let rmdirRecursiveSync
+let rmdirRecursive
+
+let hasFsRm = false
 
 module.exports = {
   atTap: false,
   get rimrafNeeded() {
     const [nodeMajor, nodeMinor] = process.versions.node.split('.').map(Number)
     /* istanbul ignore next: version specific */
+    hasFsRm = nodeMajor >= 14 && nodeMinor >= 14 || nodeMajor >= 15
```

Comment:
```suggestion
    hasFsRm = (nodeMajor === 14 && nodeMinor >= 14) || nodeMajor >= 15
```

The explicit parenthesis make this much easier to read IMO.  Also switching nodeMajor check to `=== 14`.